### ISSUE TITLE
v6.0.6 fails to build with latest XCode v10.2 (10E125)

### ISSUE THREAD

**Issue by ctscoville** [Iterable/swift-sdk#63]
I updated to the latest XCode release version 10.2 (10E125) today, and we're getting some build errors from the `Iterable-iOS-SDK` pod (v6.0.6) now when we build our project.  Can you get out a new release that fixes these errors?

It looks like the compiler is flagging the typedefs created in the IterableConstants.h file.  Since one of those has the _Nullable type applied to it, it expects that all of the other typedefs in that file have either _Nullable or _Nonnull applied to them.  So to fix this, we just modified line 24 to remove the use of _Nullable:
```
typedef void (^OnFailureHandler)(NSString *reason, NSData * data);
```

Of course the other fix would be to apply _Nullable or _Nonnull correctly in that file on all the typedefs, but I'll leave that to you to determine how you want to fix it.

**Comment by tapashmajumder** [comment]
Thanks for bringing this to our attention. We are taking a look at this issue. We expect to release a new version by the end of this week.

**Comment by tapashmajumder** [comment]
We are only seeing warnings and not errors for the issue that you mention. Maybe you have “Treat Warnings as Errors” flag set. We are still hoping to release a new version soon.

**Comment by ctscoville** [comment]
Ah, you're correct we have that set. Thanks for looking into it.  It's not blocking us, but glad to hear you may clear up those warnings in a new release. Thanks!

**Comment by mlchild** [comment]
Having this issue if compiled via carthage as well

**Comment by tapashmajumder** [comment]
We will be releasing a version to address this soon.

**Comment by tapashmajumder** [comment]
Just released 6.0.7. It should take care of the warnings.
